### PR TITLE
Fix cms fixture

### PR DESCRIPTION
**Pull request by bjrne**
The cms fixture referenced a user with db-id  `11`, and thus failed when less users were present. This PR fixes the fixture and also adds a setup script in order to make a full reset easier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,8 +8,20 @@ Originally developed as [match4healthcare](https://github.com/match4everyone/mat
...
+
+### Using docker
 - Copy `backend.prod.env.example` and `database.prod.env.example` to `backend.prod.env` and `database.prod.env` and fill in appropriate values
-- Run `docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d --build` (uses Docker), `docker-compose -f docker-compose.yml -f docker-compose.prod.yml exec backend python3 manage.py loaddata backend/cms_export.json` and visit `http://localhost:8000/`
+- Run `docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d --build` (uses Docker), `docker-compose -f docker-compose.yml -f docker-compose.prod.yml exec -it backend python3 manage.py createsuperuser`, `docker-compose -f docker-compose.yml -f docker-compose.prod.yml exec backend python3 manage.py loaddata backend/cms_export.json` and visit `http://localhost:8000/`
+
+### Manually
+- Run
+```bash
+python3 install -r backend/requirements.txt
```
Comment (Baschdl) [inline README.md]:
```suggestion
pip3 install -r backend/requirements.txt
```

File: scripts/delete_db_and_setup.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+rm backend/run/db.sqlite3
+python3 backend/manage.py migrate
+python3 backend/manage.py createsuperuser --email admin@exampleee.com --username admin
```
Comment (Baschdl) [inline scripts/delete_db_and_setup.sh]: I wouldn't use a predefined mail address, let the user enter one
```suggestion
python3 backend/manage.py createsuperuser  --username admin
```

Comment (Baschdl) [inline scripts/delete_db_and_setup.sh]:
```suggestion
python3 backend/manage.py migrate
echo "Please enter a password for the admin account:"
python3 backend/manage.py createsuperuser --email admin@exampleee.com --username admin
```

**Review by Baschdl — CHANGES_REQUESTED** [review]
and I get a casual
```
----------------------------------------
Exception happened during processing of request from ('127.0.0.1', 52276)
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.8/Frameworks/Python.framework/Versions/3.7/lib/python3.7/socketserver.py", line 650, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/local/Cellar/python/3.7.8/Frameworks/Python.framework/Versions/3.7/lib/python3.7/socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/local/Cellar/python/3.7.8/Frameworks/Python.framework/Versions/3.7/lib/python3.7/socketserver.py", line 720, in __init__
    self.handle()
  File "/usr/local/lib/python3.7/site-packages/django/core/servers/basehttp.py", line 174, in handle
    self.handle_one_request()
  File "/usr/local/lib/python3.7/site-packages/django/core/servers/basehttp.py", line 182, in handle_one_request
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/local/Cellar/python/3.7.8/Frameworks/Python.framework/Versions/3.7/lib/python3.7/socket.py", line 589, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 54] Connection reset by peer
----------------------------------------
```
but I never ran it without Docker so I don't know if this always appears

File: scripts/delete_db_and_setup.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+rm backend/run/db.sqlite3
+python3 backend/manage.py migrate
+python3 backend/manage.py createsuperuser  --username admin
```
Comment (bjrne) [inline scripts/delete_db_and_setup.sh]:
```suggestion
python3 backend/manage.py migrate
echo "Creating account with username \"admin\". "
echo "Please enter your desired email and password:"
python3 backend/manage.py createsuperuser  --username admin
```

**Review by Baschdl — APPROVED** [review]
Fine if we ignore the `ConnectionResetError`

**Pull request merged**